src/main/java/salesforce/ui/pages/AppPageFactory.java

```diff
@@ -9,23 +9,23 @@
 package salesforce.ui.pages;
 
 import salesforce.ui.pages.legalentity.*;
-
+import salesforce.ui.pages.product.*;
 import static core.config.LoadEnvironmentFile.getTheSalesforceVersion;
 
 public class AppPageFactory {
 
-    private static String version = getTheSalesforceVersion();
+    private static String skin = getTheSalesforceVersion();
 
     /**
      * Gets the legal entities page.
      *
      * @return the correct legal entities page instance.
      */
     public static LegalEntitiesPageAbstract getLegalEntitiesPage() {
-        if ("classic".equals(version)) {
-            return new LegalEntitiesClassicVersion();
+        if ("classic".equals(skin)) {
```

Comment:
I think the compare has to be skin.equals("classic")

Reply:
I think it is good as it is. Paolo once told us that it is better to put it in that way to avoid a possible error when the variable (in this case "skin") comes empty